menas/src/main/scala/za/co/absa/enceladus/menas/auth/kerberos/MenasKerberosAuthentication.scala

```diff
@@ -140,7 +137,6 @@ class MenasKerberosAuthentication extends MenasAuthentication with InitializingB
     //something here changes the log level to WARN
     auth
       .authenticationProvider(new MenasKerberosAuthenticationProvider(adServer, ldapSearchFilter, ldapSearchBase))
-      .authenticationProvider(activeDirectoryLdapAuthenticationProvider())
```

Comment:
Why is this removed?

Reply:
Spring tries to authenticate using providers in the list. And if all of them fail it will provide the exception of the last provider tried.

This leads to a situation when `MenasKerberosAuthenticationProvider` fails, Menas tries to authenticate using local LDAP and returns an exception related to that attempt, instead of the exception from `MenasKerberosAuthenticationProvider`.

I tried to authenticate using login/password and using a keytab file without `activeDirectoryLdapAuthenticationProvider` and it worked, so I thought maybe we don't need it.

If we do need it, maybe we can just swap the order of providers.